templates/taxi/manufacturer_list.html

```diff
@@ -1,33 +1,36 @@
 {% extends "base.html" %}
 
 {% block content %}
-    <h1>Manufacturer List
-    </h1>
-
-    {% if manufacturer_list %}
-      <table class="table">
+  <h1>Manufacturer List</h1>
+  
+  <a href="{% url 'taxi:manufacturer-create' %}" class="btn btn-primary mb-2">Create New Manufacturer</a>
```

Comment:
Ensure that the user has the necessary permissions to create a new manufacturer. Consider adding permission checks if they are not already implemented.